src/widget.h

```diff
@@ -134,6 +134,9 @@ template<typename T>
 class generic_factory;
 class widget;
 
+// Default label separator for widgets.
+static const std::string default_separator = "DEFAULT";
```

Comment:
This may not be the best place for this variable to exist. I would appreciate feedback on where to put this.

Reply:
If it's only used in `widget.cpp`, you should put it at the top of that file instead of the header